src/lib/data/account.js

```diff
@@ -99,9 +99,9 @@ const loadAccount = async (near, setAccount) => {
   setAccount(account);
 };
 
-export const useAccount = singletonHook(defaultAccount, () => {
+export const useAccount = (chainId) => {
   const [account, setAccount] = useState(defaultAccount);
-  const near = useNear();
+  const near = useNear(chainId);
```

Comment:
I think this should us some type of singleton as well. Similar to `useNear`, otherwise we would spawn too many effects that subscribe to updates. (Unless, I'm missing something)

Reply:
Done ✅